src/AppBundle/Helper/AutomatedEditsHelper.php

```diff
@@ -24,13 +27,23 @@ class AutomatedEditsHelper
     /** @var ContainerInterface The service container. */
     private $container;
 
+    /** @var Project $metaProject */
+    private $metaProject;
+
+    /** @var CacheItemPoolInterface The cache. */
+    protected $cache;
+
     /**
      * AutomatedEditsHelper constructor.
      * @param ContainerInterface $container
+     * @param Project|null $metaProject Solely used to mock the meta.wikimedia Project in AutomatedEditsTest
      */
-    public function __construct(ContainerInterface $container)
+    public function __construct(ContainerInterface $container, ?Project $metaProject = null)
     {
         $this->container = $container;
+        $this->cache = $container->get('cache.app');
+        $centralAuthProject = $this->container->getParameter('central_auth_project');
```

Comment:
Suggest referencing $container directly, like on line 44, instead of $this->container; same applies to the use of $this->container on line 46